### ISSUE TITLE
Milestone: Republish with resistance improvements

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#687]
For milestone: https://github.com/phetsims/circuit-construction-kit-common/milestone/3

**Comment by samreid** [comment]
Here are the shas to cherry-pick, ordered by date:

```
8327d6fa711b2db73137a4c6b8a9a5994527b024
87adaef2e8c3e16cce1b477fd88bf573fb70ce91
f9d3b7c456c769b639eb0ebfc939ae45a1f9d2d4
09727684bfb1e46175ccbcafb56d89eff7a71a2f
fb75fab9a940f8e070f859b926036c71d19c512d
ec55c81875211c3313e86f111ec2a57d6eb4d771
6c67f8746082383adb4f58ef25123d43f6dcd7d6
40351e4228712edaf73644c60366818d0e3a7357
57739633ba8557db38498aaa1fdf65ce41c3e401
c29a49d563ac61aa6abc3e4e46653710ef3a7fd1
```

**Comment by samreid** [comment]
We published DC and DC-Virtual Lab 1.2.2.  @arouinfar are there any follow-up steps?

**Comment by samreid** [comment]
Since it has been a few months, this seems safe to close.  If we discover missing parts, we can add them later.  Closing.
